docker-compose.yml

```diff
@@ -3,6 +3,8 @@ services:
   bluefin:
     build: .
     image: bluefin
+    # volumes:
```

Comment:
I added this to make development less awful so i didn't need to rebuild every time. Commented it out to make tests pass. This is super annoying.